src/dev/build/tasks/bundle_fleet_packages.ts

```diff
@@ -43,15 +35,30 @@ export const BundleFleetPackages: Task = {
     const configFilePath = config.resolveFromRepo('fleet_packages.json');
     const fleetPackages = (await read(configFilePath)) || '[]';
 
+    const parsedFleetPackages: FleetPackage[] = JSON5.parse(fleetPackages);
+
+    log.debug(
+      `Found configured bundled packages: ${parsedFleetPackages
+        .map((fleetPackage) => `${fleetPackage.name}-${fleetPackage.version}`)
+        .join(', ')}`
+    );
+
     await Promise.all(
-      JSON5.parse(fleetPackages).map(async (fleetPackage: FleetPackage) => {
+      parsedFleetPackages.map(async (fleetPackage) => {
         const archivePath = `${fleetPackage.name}-${fleetPackage.version}.zip`;
         const archiveUrl = `${eprUrl}/epr/${fleetPackage.name}/${fleetPackage.name}-${fleetPackage.version}.zip`;
 
         const destination = build.resolvePath(BUNDLED_PACKAGES_DIR, archivePath);
 
         try {
-          await downloadPackageArchive({ log, url: archiveUrl, destination });
+          await downloadToDisk({
```

Comment:
I don't see `maxAttempt` being set here, which I think is the retry count, and the default in `downloadToDisk()` is `1`.  Is it really going to retry and download if it fails?

Reply:
Whoops - you're right. Neglected to pass in `maxAttempts`. I'll pass in `3` for now since that's what we do in the `download_cloud_dependencies` build task which is similar.